ChangeLog.md

```diff
@@ -1,13 +1,28 @@
 # Change
 
-### 2022-07-09 - 5.18.0
+### 2022-xx-xx - 5.18.1
 
 | Library                                                                 | Min Version |
 | ----------------------------------------------------------------------- | ----------- |
 | `@autorest/core`                                                        | `3.8.1`     |
 | `@autorest/modelerfour`                                                 | `4.23.5`    |
 | `azure-core` dep of generated code                                      | `1.24.0`    |
-| `msrest` dep of generated code                                          | `0.7.0`    |
+| `msrest` dep of generated code                                          | `0.7.0`     |
+| `azure-mgmt-core` dep of generated code (If generating mgmt plane code) | `1.3.0`     |
+
+**Other Changes**
+
+- Optimize `--package-mode` to be compatible with more kinds of autorest configuration # 1291
+
+
+### 2022-06-09 - 5.18.0
```

Comment:
I notice `5.18.0` has been released several days ago so the date shall be `2022-06-09`?

Reply:
@msyyc I didn't actually end up releasing it until today

Reply:
@iscai-msft  I have to admit I am a little confused: Why the `5.19.0` is released with date [2022-07-13](https://github.com/Azure/autorest.python/blob/autorestv3/ChangeLog.md#2022-07-13---5190) ? Today is only `2022-06-14`